### PR TITLE
Data plan system rate

### PR DESCRIPTION
**Pull request by vthakur7f**
Fixes issue #477

### CODE REVIEW

File: systems/data-plan/api-gateway/pkg/client/dataPlan.go [hunk 1]
```diff
@@ -5,23 +5,20 @@ import (
 	"time"
```
Comment (salman-codewar) [inline systems/data-plan/api-gateway/pkg/client/dataPlan.go:5]: I think we can rename this file to `package.go`

**Review by salman-codewar — APPROVED** [review]

**Comment by salman-codewar** [comment]
Just added one small comment, Rest looks great 👍 💯

**Review by delvatt — APPROVED** [review]
LGTM!

File: systems/data-plan/api-gateway/pkg/client/dataPlan.go (hunk 1 above)
Comment (vthakur7f) [reply systems/data-plan/api-gateway/pkg/client/dataPlan.go:5]: yeah I think I missed it somehow.

**Pull request merged**